frontend/src/components/ParticipantInfo.js

```diff
@@ -229,17 +223,12 @@ const ParticipantInfo = observer(() => {
                     style={{ marginTop: 32 }}
                   >
                     <InputLabel htmlFor="user_id">UUID</InputLabel>
-                    <Input
+                    <PrevPointInput
                       id="uuid"
                       name="uuid"
-                      value={existingParticipant.pp_id}
-                      onChange={e => {
-                        participantStore.setPPId(e.target.value)
-                        participantStore.setLastFourSSN(
-                          +e.target.value.substr(2)
-                        )
-                      }}
-                      required
+                      val={existingParticipant.pp_id}
+                      changeFunc={participantStore.setPPId}
+                      changeFunc2={participantStore.setLastFourSSN}
```

Comment:
I'm -1 on `changeFunc2` the parent component should only pass in a single function that handles updating

Reply:
Yea I overlooked this a bit, the reason for this is because of this code:
```onChange={e => {
      participantStore.setPPId(e.target.value)
      participantStore.setLastFourSSN(+e.target.value.substr(2)
)```
But I can just combine those two into one action from the MobX side.

Reply:
if it makes sense the mobx fine but does it make more sense for the component to just pass in a function like 

```
const updateUUID = (e) => {
    participantStore.setPPId(e.target.value)
     participantStore.setLastFourSSN(+e.target.value.substr(2))
}

<PrevPointInput changeFunc={updateUUID} />
```